nana/modules/mega_download.py

```diff
@@ -34,30 +33,30 @@ async def subprocess_run(cmd, message):
     return talk
 
 
-@app.on_message(Filters.user(AdminSettings) & Filters.command("megadownload", Command))
+@app.on_message(filters.user(AdminSettings) & filters.command("megadownload", Command))
```

Comment:
Codacy found an issue: [mega_downloader is too complex (20) (MC0001)](https://app.codacy.com/gh/MadeByThePinsHub/Nana-Remix/pullRequest?prid=6330175)